tests/check_mlflow_lazily_imports_ml_packages.py

```diff
@@ -4,8 +4,11 @@
 
 import sys
 import importlib
+import logging
 import mlflow  # pylint: disable=unused-import
 
+logger = logging.getLogger()
```

Comment:
Let's revert the changes in this file. I already merged https://github.com/mlflow/mlflow/commit/cca1448036bba592478ab425a4d086c2981810b8.

Reply:
reverted